src/test/java/com/example/employee/controller/EmployeeControllerTest.java

```diff
@@ -28,6 +30,8 @@
 @SpringBootTest
 @ActiveProfiles("test")
 class EmployeeControllerTest {
+    @MockBean
+    StreamBridge streamBridge;
```

Comment:
Let's create some test case where we make sure the rabbitMQ message is sent out and can be received